pinot-common/src/test/java/org/apache/pinot/common/metrics/PinotJMXToPromMetricsTest.java

```diff
@@ -0,0 +1,266 @@
+    Assert.assertTrue(
+        promMetrics.contains(PromMetric.withName(exportedMetricPrefix + exportedGaugePrefix + "_" + "Value")),
+        exportedGaugePrefix);
+  }
+
+  protected void assertGaugeExportedCorrectly(String exportedGaugePrefix, List<String> labels,
+      String exportedMetricPrefix)
+      throws IOException, URISyntaxException {
+    List<PromMetric> promMetrics =
+        parseExportedPromMetrics(getExportedPromMetrics().getResponse());
+    Assert.assertTrue(promMetrics.contains(PromMetric.withNameAndLabels(
+        exportedMetricPrefix + exportedGaugePrefix + "_" + "Value", labels)));
+  }
+
+  protected void assertTimerExportedCorrectly(String exportedTimerPrefix, String exportedMetricPrefix) {
+    List<PromMetric> promMetrics = null;
+    try {
+      promMetrics = parseExportedPromMetrics(getExportedPromMetrics().getResponse());
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    } catch (URISyntaxException e) {
+      throw new RuntimeException(e);
+    }
+    for (String meterType : TIMER_TYPES) {
+      Assert.assertTrue(promMetrics.contains(PromMetric.withName(
+          exportedMetricPrefix + exportedTimerPrefix + "_" + meterType)));
+    }
+  }
+
+  protected void assertTimerExportedCorrectly(String exportedTimerPrefix, List<String> labels,
+      String exportedMetricPrefix) {
+    List<PromMetric> promMetrics;
+    try {
+      promMetrics = parseExportedPromMetrics(getExportedPromMetrics().getResponse());
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    } catch (URISyntaxException e) {
+      throw new RuntimeException(e);
+    }
+    for (String meterType : METER_TYPES) {
+      Assert.assertTrue(promMetrics.contains(PromMetric.withNameAndLabels(
+          exportedMetricPrefix + exportedTimerPrefix + "_" + meterType, labels)));
+    }
+  }
+
+  protected void assertMeterExportedCorrectly(String exportedMeterPrefix, String exportedMetricPrefix) {
+    List<PromMetric> promMetrics = null;
+    try {
+      promMetrics = parseExportedPromMetrics(getExportedPromMetrics().getResponse());
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    } catch (URISyntaxException e) {
+      throw new RuntimeException(e);
+    }
+    for (String meterType : METER_TYPES) {
+      Assert.assertTrue(promMetrics.contains(
+              PromMetric.withName(exportedMetricPrefix + exportedMeterPrefix + "_" + meterType)),
+          exportedMeterPrefix);
+    }
+  }
+
+  protected void assertMeterExportedCorrectly(String exportedMeterPrefix, List<String> labels,
+      String exportedMetricPrefix) {
+    List<PromMetric> promMetrics = null;
+    try {
+      promMetrics = parseExportedPromMetrics(getExportedPromMetrics().getResponse());
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    } catch (URISyntaxException e) {
+      throw new RuntimeException(e);
```

Comment:
nit:
```suggestion
    } catch (IOException | URISyntaxException e) {
      throw new RuntimeException(e);
```

Reply:
Wrapped everything is a single exception block.